app/Http/Controllers/User/ProfileController.php

```diff
@@ -142,12 +139,12 @@ public function index ()
         // Todo - Store this metadata in Redis
         $totalPhotosAllUsers = Photo::count();
         // Todo - Store this metadata in Redis
-        $totalLitterAllUsers = Photo::sum('total_litter');
+        $totalTagsAllUsers = Photo::sum('total_litter'); // this doesn't include brands
```

Comment:
We also need to store this on Redis

Reply:
True, I noticed the Todo there, but I wanted to keep this PR simple and fix the issues with the current codebase. Migrating to Redis would introduce some complexity which I'm not sure we can handle without its own PR.